src/main/java/com/aetherteam/aether_genesis/event/listeners/EntityListener.java

```diff
@@ -11,15 +13,17 @@
 public class EntityListener {
     @SubscribeEvent
     public static void DisabledNightSpawn(MobSpawnEvent.FinalizeSpawn event){
-        LivingEntity zephyr = event.getEntity();
-        if(zephyr instanceof Zephyr && !zephyr.getLevel().isClientSide && zephyr.getLevel().isNight())
+        LivingEntity entity = event.getEntity();
+        if(entity.getType() == AetherEntityTypes.ZEPHYR.get() && !entity.getLevel().isClientSide && entity.getLevel().isNight())
+            event.setSpawnCancelled(true);
+        if(entity.getType() == GenesisEntityTypes.TEMPEST.get() && !entity.getLevel().isClientSide && !entity.getLevel().isNight())
```

Comment:
The tempest spawn handling should already be handled in the tempest class.

Reply:
ah, ok